tests/Main.hs

```diff
@@ -94,17 +94,16 @@ docTests = testCase "doctests" $ do
   putStrLn "Running all doctests..."
   Test.DocTest.doctest
     [ "--verbose"
-    , "-package"
-    , "ghc"
+    , "-package", "ghc-lib-parser"
     , "-isrc"
 #ifdef SWAGGER
+    , "-DSWAGGER"
+#endif
 #ifdef SWAGGER_WRAPPER_FORMAT
-    , "-isrc/swagger-wrapper-format"
     , "-DSWAGGER_WRAPPER_FORMAT"
+    , "-isrc/swagger-wrapper-format"
 #else
     , "-isrc/no-swagger-wrapper-format"
-#endif
-    , "-DSWAGGER"
 #endif
```

Comment:
I would rather keep the `#ifdef SWAGGER_WRAPPER_FORMAT` within the `#ifdef SWAGGER` to make it clear that the code only makes sense to include when both macros are defined.  Though in practice people should not define `SWAGGER_WRAPPER_FORMAT` on its own, without `SWAGGER`, mistakes could happen and it would be nice for the intent to be clear here.

Reply:
My preference here is a pretty minor issue, so I'll merge without resolving this thread.

Reply:
I initially made these Swagger-related fixes thinking they would resolve build dependency issues with different GHC versions, but after further consideration, I realized they were unrelated. I'll delete this entire commit.